docs/Concepts/Profiling.md

```diff
@@ -1,13 +1,13 @@
-# Quorum Profiling
+# Quorum profiling
 
-[Quorum Profiling](https://github.com/ConsenSys/quorum-profiling) is a custom toolset used to benchmark
+[Quorum profiling](https://github.com/ConsenSys/quorum-profiling) is a custom toolset used to benchmark
 transaction throughput and network statistics on any existing Quorum network using the
 [JMeter](https://github.com/ConsenSys/quorum-profiling/tree/master/jmeter-test) and
 [TPS monitoring](https://github.com/ConsenSys/quorum-profiling/tree/master/tps-monitor) tool profiles.
 Alternatively, it could also be used to spin up an entire GoQuorum network from scratch in AWS and benchmark
 the network for TPS, CPU/Memory usage. The various scenarios of usage is explained [here](https://github.com/ConsenSys/quorum-profiling)
 
-## Metrics Visualisation
+## Metrics visualisation
```

Comment:
```suggestion
## Metrics visualization
```

[Fix non-US to US spelling](https://docs.microsoft.com/en-us/style-guide/word-choice/use-us-spelling-avoid-non-english-words) when you see it.